cmd/image-builder-maintenance/config.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"strconv"
+)
+
+// Do not write this config to logs or stdout, it contains secrets!
+type Config struct {
+	DryRun                bool   `env:"DRY_RUN"`
+	EnableDBMaintenance   bool   `env:"ENABLE_DB_MAINTENANCE"`
+	ClonesRetentionMonths int    `env:"DB_CLONES_RETENTION_MONTHS"`
+	PGHost                string `env:"PGHOST"`
+	PGPort                string `env:"PGPORT"`
+	PGDatabase            string `env:"PGDATABASE"`
+	PGUser                string `env:"PGUSER"`
+	PGPassword            string `env:"PGPASSWORD"`
+	PGSSLMode             string `env:"PGSSLMODE"`
+}
+
+// *string means the value is not required
+// string means the value is required and should have a default value
+func LoadConfigFromEnv(intf interface{}) error {
```

Comment:
We could change `internal/config`'s LoadConfigFromEnv to accept either this config or the ImageBuilderConfig by moving the clowder stuff into a separate function, as the clowder stuff should be applied afterwards anyway.

Reply:
Thanks, for pointing this out. That's a good idea for sure!

Reply:
let's do it in a followup!